ConsoleApp2/Program.cs

```diff
@@ -160,10 +160,35 @@ public static int EnterNumber()
                     return value;
                 }
                 Console.WriteLine("Please, enter integer value");
-            }           
-
+            }
         }
-
+        public static float EnterText()
+        {
+            float value;
+            while (true)
+            {
+                var answer = Console.ReadLine();
+                if (float.TryParse(answer, out value))
+                {
+                    return value;
+                }
+                Console.WriteLine("Please, enter integer value");
```

Comment:
It asks to enter "integer" value

Reply:
i don't understand

Reply:
You need to enter float value. But asking user to enter "integer".